<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginAdvComponent</class>
 <widget class="QFrame" name="CLoginAdvComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login component</string>
  </property>
  <layout class="QVBoxLayout" name="hl_LoginAdvComponent">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_Network">
     <property name="title">
      <string>Network</string>
     </property>
     <layout class="QVBoxLayout" name="vl_Network">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="BlackGui::Components::CNetworkDetailsComponent" name="comp_NetworkDetails">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_MatchingLog">
     <property name="title">
      <string>Matching log.</string>
     </property>
     <layout class="QVBoxLayout" name="vl_MatchingLogConnected">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="BlackGui::Components::CModelMatcherLogEnable" name="comp_MatchingLog">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_PilotsDetails">
     <property name="title">
      <string>Pilot's info</string>
     </property>
     <layout class="QVBoxLayout" name="vl_PilotIngoGroupBox">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="BlackGui::Editors::CPilotForm" name="form_Pilot">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_OwnAircraft">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="title">
      <string>Own aircraft</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="vl_OwnAircraft">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="BlackGui::Components::COwnAircraftComponent" name="comp_OwnAircraft">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="vs_LoginAdvComponent">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QFrame" name="fr_Buttons">
     <layout class="QGridLayout" name="gl_Buttons">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="1" column="1">
       <spacer name="hs_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pb_Cancel">
        <property name="text">
         <string>cancel</string>
        </property>
        <property name="shortcut">
         <string>Esc</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pb_Connect">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CModelMatcherLogEnable</class>
   <extends>QFrame</extends>
   <header>blackgui/components/modelmatcherlogenable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CPilotForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/pilotform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CNetworkDetailsComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/networkdetailscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::COwnAircraftComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/ownaircraftcomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>form_Pilot</tabstop>
  <tabstop>pb_Connect</tabstop>
  <tabstop>pb_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
